test/e2e/spec/shared_spec.rb

```diff
@@ -0,0 +1,257 @@
+RSpec.describe CardanoWallet::Shared do
+
+  describe CardanoWallet::Shared::Wallets do
+
+    before(:each) do
+      teardown
+    end
+
+
+    describe "Create wallets" do
+      it "I can create, get and delete wallet from mnemonics getting acc_xpub from cardano-address" do
+        w = SHARED.wallets
+
+        m24 = mnemonic_sentence(24)
+        acc_ix = '0H'
+        acc_xpub = cardano_address_get_acc_xpub(m24, "1852H/1815H/#{acc_ix}")
```

Comment:
it is going to be 1854H very soon - just indicating

Reply:
Yep. :+1: